### PR TITLE
Add cgroups v1 periodic jobs for 4.14

### PR DESCRIPTION
**Pull request by harche**
With cgroups v2 getting enabled by default in 4.14, it would be good to have cgroups v1 periodic jobs for 4.14 

PR to enable cgroups v2 by default in 4.14 - https://github.com/openshift/machine-config-operator/pull/3789

### CODE REVIEW

**Comment by haircommander** [comment]
nit: description and commit are out of date (they specifically list v2)

**Comment by harche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-ci-4.14-e2e-aws-cgroupsv1

**Comment by harche** [comment]
/pj-rehearse  periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-cgroupsv1

**Comment by harche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-ci-4.14-e2e-aws-cgroupsv1

**Comment by harche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-azure-cgroupsv1 periodic-ci-openshift-release-master-ci-4.14-e2e-aws-cgroupsv1 periodic-ci-openshift-release-master-nightly-4.14-e2e-vsphere-cgroupsv1 periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-cgroupsv1 periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-cgroupsv1 periodic-ci-openshift-release-master-ci-4.14-e2e-azure-cgroupsv1 periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-aws-cgroupsv1

**Comment by harche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-ci-4.13-e2e-azure-cgroupsv2 periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-cgroupsv2 periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-cgroupsv2 periodic-ci-openshift-release-master-nightly-4.13-e2e-aws-cgroupsv2 periodic-ci-openshift-hypershift-release-4.13-periodics-e2e-aws-cgroupsv2 periodic-ci-openshift-release-master-nightly-4.13-e2e-azure-cgroupsv2

**Comment by harche** [comment]
/pj-rehearse ci/rehearse/periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-aws-cgroupsv1 ci/rehearse/periodic-ci-openshift-release-master-ci-4.14-e2e-aws-cgroupsv1 ci/rehearse/periodic-ci-openshift-release-master-ci-4.14-e2e-azure-cgroupsv1 ci/rehearse/periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-cgroupsv1 ci/rehearse/periodic-ci-openshift-release-master-nightly-4.14-e2e-azure-cgroupsv1 ci/rehearse/periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-cgroupsv1

**Comment by harche** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-aws-cgroupsv1 periodic-ci-openshift-release-master-ci-4.14-e2e-aws-cgroupsv1 periodic-ci-openshift-release-master-ci-4.14-e2e-azure-cgroupsv1 periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-cgroupsv1 periodic-ci-openshift-release-master-nightly-4.14-e2e-azure-cgroupsv1 periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-cgroupsv1

**Comment by harche** [comment]
I verified the journal logs of cgroup v1 and cgroup v2. The nodes are configured correctly with the intended cgroup version for that job. 

The occasional flakiness we see is not due to cgroup version, jobs with nightly and CI builds tend to be more flaky than regular builds.

**Comment by harche** [comment]
cc @technical-release-team-approvers @sjenning @wking

**Comment by haircommander** [comment]
/lgtm

from node team perspective

**Comment by harche** [comment]
> the output of the hypershift test implies there's something misconfigured about it

@sjenning @wking do you spot anything missing or incorrect in hypershift workflow? https://github.com/openshift/release/pull/41190/files#diff-f79c41eb17645336e7f75e9bc7eb2e02e65162f472564341f0ff7976657cbcc2

**Comment by harche** [comment]
Let me copy the[ workflow of the conformance job](https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml ) and see if that helps.

**Comment by harche** [comment]
> > the output of the hypershift test implies there's something misconfigured about it
> 
> @sjenning @wking do you spot anything missing or incorrect in hypershift workflow? https://github.com/openshift/release/pull/41190/files#diff-f79c41eb17645336e7f75e9bc7eb2e02e65162f472564341f0ff7976657cbcc2

Let me try the [workflow of the conformance job](https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/aws/conformance/hypershift-aws-conformance-workflow.yaml#L26) and see if it helps.

**Comment by harche** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-aws-cgroupsv1

**Comment by harche** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-cgroupsv1

**Review by wking — APPROVED** [review]
/lgtm

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml [hunk 1]
```diff
@@ -279,6 +279,15 @@ tests:
       enable:
       - observers-resource-watch
     workflow: baremetalds-e2e-ovn-ipv6
+- as: e2e-metal-cgroupsv1
```
Comment (stbenjam) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:282]: Metal jobs use the e2e-metal-ipi designation

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml [hunk 2]
```diff
@@ -410,6 +419,14 @@ tests:
   steps:
     cluster_profile: vsphere-multizone
     workflow: openshift-e2e-vsphere-upi-zones
+- as: e2e-vsphere-cgroupsv1
```
Comment (stbenjam) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:422]: Ditto, vsphere jobs use e2e-vsphere-ipi, our naming hasn't always been the most consistent but please try to match what's in use along with https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml (hunk 1 above)
Comment (stbenjam) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:282]: Also the CNI provider is a required component of job names

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml (hunk 2 above)
Comment (stbenjam) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:422]: e2e-vsphere-ipi-ovn-cgroupsv1

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml [hunk 3]
```diff
@@ -668,6 +685,16 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-azurestack-upi-csi
+- as: e2e-azure-cgroupsv1
```
Comment (stbenjam) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:688]: e2e-azure-ovn-cgroupsv1

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml [hunk 4]
```diff
@@ -678,6 +705,16 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-sdn-cgroupsv2
+- as: e2e-aws-cgroupsv1
```
Comment (stbenjam) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:708]: e2e-aws-ovn-cgroupsv1

File: ci-operator/step-registry/openshift/e2e/azure/cgroupsv2/openshift-e2e-azure-cgroupsv2-workflow.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+workflow:
+  as: openshift-e2e-azure-cgroupsv2
```
Comment (stbenjam) [inline ci-operator/step-registry/openshift/e2e/azure/cgroupsv2/openshift-e2e-azure-cgroupsv2-workflow.yaml:2]: Is this used anywhere?  I don't see it mentioned elsewhere in this PR

Comment (harche) [reply ci-operator/step-registry/openshift/e2e/azure/cgroupsv2/openshift-e2e-azure-cgroupsv2-workflow.yaml:2]: Thanks for pointing that out, I will remove these files.

**Comment by stbenjam** [comment]
/lgtm

Next step is you'll also want to add these as informers after these have been running for a bit https://docs.ci.openshift.org/docs/architecture/release-gating/#becoming-informing

I think we can waive the one sprint thing for these

**Comment by harche** [comment]
> /lgtm
> 
> Next step is you'll also want to add these as informers after these have been running for a bit https://docs.ci.openshift.org/docs/architecture/release-gating/#becoming-informing
> 
> I think we can waive the one sprint thing for these

Sure, that makes sense. Thanks for your review.

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-dualstack

File: ci-operator/step-registry/openshift/manifests/cgroupsv1/openshift-manifests-cgroupsv1-commands.sh [hunk 6]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
Comment (EmilienM) [inline ci-operator/step-registry/openshift/manifests/cgroupsv1/openshift-manifests-cgroupsv1-commands.sh:1]: FYI I'm proposing https://github.com/openshift/release/pull/42478 which modifies these manifests to use Node API.

**Pull request merged**